openapi/v2/change/release_package.yml

```diff
@@ -0,0 +1,328 @@
+---
+openapi: 3.0.0
+info:
+  version: '3.0'
+  title: Release Package REST API
+  license:
+    name: MIT
+servers:
+- url: http://{orgId}.salesforce.com/services/apexrest/{namespace}/v3
+  variables:
+    namespace: 
+      default: PDLM
+      description: This value should default to PDLM for production environments. 
+        For dev environments, provide the namespace to your dev org.
+paths:
+  '/change/{changeId}':
+    get:
+      summary: Change (Release Package)
+      description: Get a change order's details along with related information such
+        as Affected Items, Bills of Materials, Approved Manufacturers Lists, and Attachments.
+        This endpoint is useful for integrations with downstream systems such as ERP.
+      operationId: getChangeOrder
+      tags:
+      - change
+      parameters:
+      - name: changeId
+        in: path
+        description: Change Id Or Change Number
+        required: true
+        schema:
+          type: string
+          format: string
+      - name: affectedItems
+        in: query
+        description: Include affected items including details fields and costing information.
+        required: false
+        schema:
+          type: boolean
+          format: boolean
+          default: false
+      - name: bom
+        in: query
+        description: Include the BOM for each affected item in the payload, only includes the
+          first level. affectedItems parameter must be true.
+        required: false
+        schema:
+          type: boolean
+          format: boolean
+          default: false
+      - name: amls
+        in: query
+        description: Include AMLs for each affected item in the payload. affectedItems
+          parameter must be true.
+        required: false
+        schema:
+          type: boolean
+          format: boolean
+          default: false
+      - name: attachments
+        in: query
+        description: Include information about Change attachments. If affectedItems
+          is true, Item attachments will also be included.
+        required: false
+        schema:
+          type: boolean
+          format: boolean
+          default: false
+      - name: useXMLexportFieldset
+        in: query
+        description: Use the XML Export field set to determine which fields will
+          be returned in the response. Default is false to return all fields.
+        required: false
+        schema:
+          type: boolean
+          format: boolean
+          default: false
+      - name: limit
+        in: query
+        description: Used to limit the number of affected items included in the response
+        required: false
+        schema:
+          type: integer
+          format: integer
+          default: 0
+      - name: page
```

Comment:
If I make the first call with limit=100, wouldn't the next page need params of `limit=100&page=2`.

Is this much different than `limit=100&offset=100`?

Reply:
Same thing, we can use either. whichever way, we will generate what the 2nd,3rd,... url will be in the 'next' parameter response. Whichever param you think is better.